package.json

```diff
@@ -35,7 +35,7 @@
     "gatsby-transformer-sharp": "^2.5.3",
     "graphql": "^15.1.0",
     "isomorphic-fetch": "^2.2.1",
-    "particlesjs": "^2.2.3",
+    "particles.js": "^2.0.0",
```

Comment:
Have you ever tried tsParticles? https://github.com/matteobruni/tsparticles

Reply:
Whoa, never knew this existed. I’ll definitely check it out. 👀